openwrt-openwisp-monitoring/files/monitoring.init

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+STOP=15
+USE_PROCD=1
+PROG="/usr/sbin/openwisp_monitoring"
+PROG_NAME="OpenWISP monitoring daemon"
+
+start_service() {
+		config_load openwisp
+		base_url=$(config_get http url)
+		uuid=$(config_get http uuid)
+		key=$(config_get http key)
+		verify_ssl=$(config_get http verify_ssl)
```

Comment:
Try using [`config_get_bool`](https://openwrt.org/docs/guide-developer/config-scripting#reading_booleans) for `verify_ssl` and any other booleans

Reply:
Have a look at my pullrequest #108 for parsing the uci to generate the service arguments. This is much cleaner

Reply:
Sorry, which pull request is referred here?